core/storage/exploration/gae_models.py

```diff
@@ -225,6 +211,32 @@ def delete_multi(
                 commit_log_models)
             datastore_services.put_multi_async(commit_log_models)
 
+    def _reconstitute(self, snapshot_dict):
+        """Populates the model instance with the snapshot.
+        Some old ExplorationSnapshotContentModels can contain fields
+        and field values that are no longer supported and would cause
+        an exception when we try to reconstitute a ExplorationModel from
+        them. We need to remove or replace these fields and values.
+
+        Args:
+            snapshot_dict: dict(str, *). The snapshot with the model
+                property values.
+
+        Returns:
+            VersionedModel. The instance of the VersionedModel class populated
+            with the snapshot.
+        """
+
+        if 'skill_tags' in snapshot_dict:
+            del snapshot_dict['skill_tags']
+        if 'default_skin' in snapshot_dict:
+            del snapshot_dict['default_skin']
+        if 'skin_customizations' in snapshot_dict:
+            del snapshot_dict['skin_customizations']
```

Comment:
You will probably need tests for this code.

Reply:
done